js/data/odata/context.js

```diff
@@ -109,27 +106,26 @@ const ODataContext = Class.inherit({
      * @param3 httpMethod:object
      * @return Promise<void>
      */
-    invoke: function(operationName, params, httpMethod) {
-        params = params || {};
-        httpMethod = (httpMethod || 'POST').toLowerCase();
+    invoke(operationName, params = {}, httpMethod = 'POST') {
```

Comment:
Can I call `invoke('name', null, null)` or  `invoke('name', void 0, void 0)`?

Reply:
You can call `invoke('name', void 0, void 0)`, `invoke('name', undefined, undefined)` or `invoke('name')` and it will be work correcrtly.
But it isn't work with `invoke('name', null, null)`. Do we have a cases, when we call it with `null`?